client/src/components/CreateCard/Comment.js

```diff
@@ -3,11 +3,19 @@ import ChatBubbleOutlineOutlinedIcon from "@material-ui/icons/ChatBubbleOutlineO
 import { TextField, Grid, Typography } from "@material-ui/core";
 import BlueButton from "../BlueButton";
 import { CardContext } from "./cardContext";
+import { useHistory } from "react-router";
```

Comment:
not react-router-dom?

Reply:
What's the different between the two?

Reply:
that was a mistake. shouldve been react router dom. im not exactly sure the difference but it works either way i guess. will fix.

Reply:
Ok, looks good other than that